app/DoctrineMigrations/Version20141208223436.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace Application\Migrations;
+
+use Doctrine\DBAL\Migrations\AbstractMigration;
+use Doctrine\DBAL\Schema\Schema;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+class Version20141208223436 extends AbstractMigration
```

Comment:
this migration doesn't refer to the functionality of your project (and can be removed)